search-algos/exponential_search/exponential_search.c

```diff
@@ -0,0 +1,84 @@
+EXPONENTIAL SEARCH in C:
+        This search algo for searching find its application in large sized arrays.It is better than Linear and
+        Jump search and equivalent to Binary Search.Its time complexity is O(logn).The working of the search is
+        as follows:
+        1. Firstly,it finds the range where element to be searched can be present.
+        2. Next,it does Binary Search across the found range.
+---------------------------------------------------------------------------------------------------------------------------------------------
+#include <stdio.h>
+#include <stdlib.h>
+#include<math.h>                //including preprocessor directives
+#define max_len 500             //defining the value of max_len as 500.
+int a[max_len],num;             //declaring global variables
+
+int min(int i,int j)
+{
+    if(j>i)
+         return i;
+    else
+         return j;
+}
+int binary_search(int number,int beg,int end)   /*binary search function implementing its algo and
+{                                                   finding element in the given range.
```

Comment:
why this code is commented?

Reply:
Sorry,added comments afterwards.Forgot to close.Updating it now.